pipeline/inputs/kafka.md

```diff
@@ -15,6 +15,7 @@ This plugin uses the official [librdkafka C library](https://github.com/edenhill
 | poll\_ms | Kafka brokers polling interval in milliseconds. | 500 |
 | Buffer\_Max\_Size | Specify the maximum size of buffer per cycle to poll kafka messages from subscribed topics. To increase throughput, specify larger size. | 4M |
 | poll\_ms | Kafka brokers polling interval in milliseconds. | 500 |
+| async_commit | Enable Kafka commit operation asynchronously. This option takes a boolean value: True/False. | False |
```

Comment:
```suggestion
| async_commit | Enable Kafka commit operation asynchronously. This parameter accepts a boolean value of `true` or `false`. | `false` |
```

Reply:
@lockewritesdocs 
Will incorporate your suggestion. Thanks for the review👍